.travis.yml

```diff
@@ -3,8 +3,12 @@ language: java
 dist: trusty
 sudo: required
 
+os:
+- linux
+- osx
+
 jdk:
-- oraclejdk8
+- openjdk11
```

Comment:
Do we keep travis build on Java8 or activate Java8 and Java11 build?

Reply:
Also, do we use oraclejdk11 or openjdk11 on travis ? 

In the image there is openjdk7 openjdk8 oraclejdk8 (default) oraclejdk9 oraclejdk11 but to use openjdk11 it calls install-jdk.sh (excerpt a build: 
```
Installing openjdk11
$ export JAVA_HOME=~/openjdk11
$ export PATH="$JAVA_HOME/bin:$PATH"
$ ~/bin/install-jdk.sh --target "/home/travis/openjdk11" --workspace "/home/travis/.cache/install-jdk" --feature "11" --license "GPL" --cacerts
install-jdk.sh 2019-05-02
Variables:
  feature = 11
  license = GPL
       os = linux-x64
      url = https://download.java.net/java/GA/jdk11/9/GPL/openjdk-11.0.2_linux-x64_bin.tar.gz
   status = 200
  archive = /home/travis/.cache/install-jdk/openjdk-11.0.2_linux-x64_bin.tar.gz
Downloading JDK from https://download.java.net/java/GA/jdk11/9/GPL/openjdk-11.0.2_linux-x64_bin.tar.gz...
openjdk version "11.0.2" 2019-01-15
OpenJDK Runtime Environment 18.9 (build 11.0.2+9)
OpenJDK 64-Bit Server VM 18.9 (build 11.0.2+9, mixed mode)
```
)

We should check how much it speeds up the build to use oraclejdk11 instead of openjdk11

Reply:
openjdk11